### ISSUE TITLE
ERROR: Unable to extract uploader id;

### ISSUE THREAD

**Issue by AragCF** [ytdl-org/youtube-dl#31579]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2021.12.17**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
e:\54\Trainings\[Павел Кочкин]\Chatbot>CALL "youtube-dl.exe" https://www.youtube.com/watch?v=xmnSwaiYN5M -f (mp4) -i --write-auto-sub --sub-lang ru,en --write-info-json --write-annotations --write-description --geo-bypass --verbose
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['https://www.youtube.com/watch?v=xmnSwaiYN5M', '-f', '(mp4)', '-i', '--write-auto-sub', '--sub-lang', 'ru,en', '--write-info-json', '--write-annotations', '--write-description', '--geo-bypass', '--verbose']
[debug] Encodings: locale cp1251, fs mbcs, out cp866, pref cp1251
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.19041
[debug] exe versions: ffmpeg n4.4-6-g7e9b9f24df, ffprobe n4.4-6-g7e9b9f24df
[debug] Proxy map: {}
[youtube] xmnSwaiYN5M: Downloading webpage
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\youtube.py", line 1794, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 1012, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.

```

## Description

Today I noticed that the following download fails. Also other downloads also fails with the same error.

**Comment by gspears333** [comment]
This issue also hit me today, 1st time ever I've had a youtube-dl bug.  I have verified my installation is current via the youtube-dl -U switch.   It occurred when trying to download youtube vid ( https://www.youtube.com/watch?v=Jnc77gMihPo  ) on my Windows 10 OS HP laptop.  Verbose log for you -- thanks and kind regards:

F:\SETUP\Programs\HP_6570b>youtube-dl  --verbose https://www.youtube.com/watch?v=Jnc77gMihPo
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['--verbose', 'https://www.youtube.com/watch?v=Jnc77gMihPo']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2021.12.17
[debug] Python version 3.4.4 (CPython) - Windows-10-10.0.19041
[debug] exe versions: none
[debug] Proxy map: {}
[youtube] Jnc77gMihPo: Downloading webpage
ERROR: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 815, in wrapper
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\YoutubeDL.py", line 836, in __extract_info
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 534, in extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\youtube.py", line 1794, in _real_extract
  File "C:\Users\dst\AppData\Roaming\Build archive\youtube-dl\ytdl-org\tmpupik7c6w\build\youtube_dl\extractor\common.py", line 1012, in _search_regex
youtube_dl.utils.RegexNotFoundError: Unable to extract uploader id; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; type  youtube-dl -U  to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.


F:\SETUP\Programs\HP_6570b>

**Comment by AragCF** [comment]
Now I wonder how to compile youtube-dl from sources on my Windows 10.
Is there a convenient way to make exe file? Or somebody to provide the compiled exe of dev build?

**Comment by elgonzo** [comment]
> [x]  I've searched the bugtracker for similar bug reports including closed ones

No, you didn't. Quite obviously not. But you couldn't resist ticking that checkbox, couldn't you...?

**Comment by AragCF** [comment]
> > [x]  I've searched the bugtracker for similar bug reports including closed ones
> 
> No, you didn't. Quite obviously not. But you couldn't resist ticking that checkbox, couldn't you...?

No, I did. I found quite similar issue but thought that it's caused by some different reasons. Sorry if I am wrong.

**Comment by atsen-dev** [comment]
I've same issue, and It was working good 2 days ago...
Anyone know what is the problem ?

**Comment by AragCF** [comment]
> I've same issue, and It was working good 2 days ago... Anyone know what is the problem ?

Looks like it's fixed but not released yet.
https://github.com/ytdl-org/youtube-dl/issues/31577#issuecomment-1435695483

As far as I see the problem is youtube changed some output format that youtube-dl can not handle properly.

**Comment by gspears333** [comment]
All:  the team has created a patch/interim fix available here:  https://github.com/ytdl-patched/youtube-dl/releases/tag/2023.02.18.1919

I understand that for some of you, English is a second language.  Just please always do your best in reporting issues because the dev team works hard and dealing with the actual issues is hard enough.

**Comment by dirkf** [comment]
Duplicate of #31530.

**Comment by AragCF** [comment]
> All: the team has created a patch/interim fix available here: https://github.com/ytdl-patched/youtube-dl/releases/tag/2023.02.18.1919
> 
> I understand that for some of you, English is a second language. Just please always do your best in reporting issues because the dev team works hard and dealing with the actual issues is hard enough.
> 
> EDIT: These are my opinions. I am not affiliated with the dev team and i do not speak for them -- thanks.

Yeah Okay.
